client/sites-dashboard/components/sites-grid-tile.tsx

```diff
@@ -6,6 +6,23 @@ const container = css( {
 	width: '100%',
 	flexDirection: 'column',
 	minWidth: 0,
+	transition: 'all 0.3s ease-in-out',
+	':hover': {
+		transform: 'scale(1.07, 1.07)',
+		':after': {
+			//opacity: 1,
```

Comment:
uncomment the opacity to apply the shadow. To see what it looks like with and without a shadow